product_variant_company/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module extend Odoo functionalities of the "product.product" model
+by adding a related field that provides information about
+the associated company for each product.
```

Comment:
One last thing: can you please add here the explanation that is a technical module and does not change anything in the std behavior (see related comment).